source/Calamari/Calamari.csproj

```diff
@@ -0,0 +1,127 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.0.0.0</Version>
+    <TargetFrameworks>net452;netcoreapp1.0</TargetFrameworks>
```

Comment:
Tentacle 3.0 can run on 2003 (https://octopus.com/blog/octopus-3.1-will-be-.net-4.5), and people are doing that with the latest version of OD server (https://octopusdeploy.slack.com/archives/C072C2D5L/p1470097608000604)